open-search/oci/opensearch-upload.py

```
@@ -50,4 +61,4 @@
 url = f"{OPENSEARCH_URL}/{INDEX_NAME}/_doc"
 response = requests.post(url, json=all, auth=(USERNAME, PASSWORD), verify=False)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)